images/k8s-cloud-builder/variants.yaml

```diff
@@ -1,20 +1,20 @@
 variants:
   v1.32-cross1.23-bullseye:
     CONFIG: 'cross1.23'
-    KUBE_CROSS_VERSION: 'v1.32.0-go1.23.0-bullseye.0'
+    KUBE_CROSS_VERSION: 'v1.32.0-go1.23.1-bullseye.0'
   v1.31-cross1.23-bullseye:
     CONFIG: 'cross1.23'
-    KUBE_CROSS_VERSION: 'v1.31.0-go1.23.0-bullseye.0'
+    KUBE_CROSS_VERSION: 'v1.31.0-go1.23.1-bullseye.0'
   # TODO: remove this after upgrade to Go 1.23 is finalized and there's on risk of reverting
```

Comment:
when shall we remove this? from release notes, 1.31.1 is still build with go 1.22.6, I think it's not finalized yet.

Reply:
@cpanato @xmudrii